rnc2rng/parser.py

```diff
@@ -50,7 +50,8 @@ def lex(src):
     'AMP', 'CNAME', 'COMBINE', 'COMMA', 'DOCUMENTATION', 'EQUAL', 'ID',
     'LBRACE', 'LBRACKET', 'LPAREN', 'LIST', 'LITERAL', 'MINUS', 'MIXED',
     'PLUS', 'PIPE', 'QMARK', 'RBRACE', 'RBRACKET', 'RPAREN', 'STAR', 'TILDE',
-] + [s.upper() for s in KEYWORDS])
+] + [s.upper() for s in KEYWORDS], precedence=[("left", ['TILDE'])])
```

Comment:
Is the explicit precedence here actually needed?

Reply:
Yes, I think so.  Although I'm not sure of a better way to avoid a conflict.